mmdet3d/models/dense_heads/centerpoint_head.py

```diff
@@ -327,6 +327,8 @@ def __init__(self,
                 in_channels=share_conv_channel, heads=heads, num_cls=num_cls)
             self.task_heads.append(builder.build_head(separate_head))
 
+        self.with_velocity = 'vel' in common_heads.keys()
```

Comment:
This can be written as a property of the class CenterPoint

Reply:
If so, some checking codes would be better to be be added to avoid conflicts in the config file.

Reply:
I mean you can implement this line in a way like [this](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/models/detectors/mvx_two_stage.py#L105)? We can keep the current logic to assign its value.

Reply:
Good suggestion. I added it in the new commit.